src/clients/migration_proposals.rb

```diff
@@ -0,0 +1,9 @@
+require "migration/proposal_store"
+require "installation/proposal_runner"
+require "yast"
+
+Yast.import "Wizard"
+
+# Proposal runner expect already opened wizard dialog
+Yast::Wizard.OpenNextBackDialog
```

Comment:
Um, a new dialog is open here, but I cannot see anything like `Wizard.CloseDialog` here, looks like UI leak for me...